myob/endpoints.py

```diff
@@ -31,7 +31,10 @@
     'Sale/CustomerPayment/': {
         'name': 'customer_payments',
         'methods': [
-            (ALL, '', 'sale customer payment type')
+            (ALL,    '', 'sale customer payment type'),
```

Comment:
Do you mind fixing up this line as well while we're here?
```suggestion
            (ALL,    '', 'sale customer payment'),
```

Reply:
Done!